src/bin/bat/clap_app.rs

```diff
@@ -422,11 +433,17 @@ pub fn build_app(interactive_output: bool) -> ClapApp<'static, 'static> {
                      '--style=\"..\"' option to the configuration file or export the \
                      BAT_STYLE environment variable (e.g.: export BAT_STYLE=\"..\").\n\n\
                      Possible values:\n\n  \
-                     * full: enables all available components (default).\n  \
```

Comment:
Here we have another high level problem. We don't want to make this `auto` by default. But  we don't want `full` as default either since we don't want to show all header fields by default.

So we have two choices now:
1. We only add `header-filename` and `header-filesize` in this PR so we can keep `full` as default, which in turn allows us to postpone this problem to a follow-up PR (this is the option I prefer)
2. We deal with this problem in this PR. Maybe we should add a new variant `recommended` that is default? Or maybe just call the default `default`?

I would like to ask @sharkdp and @keith-hall: Are you OK with starting to show both `header-filename` and `header-filesize` by default? If not, then we might as well go with option 2 right away and skip a potential follow-up PR.

Reply:
I'm okay with it :)